example/REQUIRE

```diff
@@ -0,0 +1 @@
+ConjugatePriors
```

Comment:
This kind of sub-directory dependency is for example evaluated for test directories